packages/trader/test/UniswapV3Trader.ts

```diff
@@ -0,0 +1,427 @@
+import { web3, assert } from "hardhat";
```

Comment:
These tests are very similar to the `UniswapV2Trader` tests in that they take the contract through a number of different cases and ensures the bot moves the market as expected.